### DISCUSSION TITLE
Making Flockdrone abilities more accessible

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#226] [Ideas]
Discussion for making Flockdrone abilities more accessible. Discussion version of #12.

Needed since there's really quite a bit of stuff you can do with Flockdrones that depends on intent.

Could maybe be fixed by adding an ability icon for Flockdrones that when clicked opens a hint menu telling you what you can do, or gives a message in the chat.

Alternatively, there's an argument that you could just look at the wiki for this information. Could reduce number of things on hud if the ability icon is the best alternative.

**Comment by stonepillars** [comment]
It might be one of those 'read the wiki' things.
Something that I don't like particularly but could be a solution is showing a context menu when using the converter on flock stuff. That's a pretty radical change from how it works now, though.

**Reply by amylizzle** [reply]
That would suck for caging though

**Reply by stonepillars** [reply]
Would it be reasonable to assume they'd understand some basic ideas behind the intents? i.e. HARM being cage, HELP being convert? The only weird one out is making a barricade with DISARM intent.

**Reply by TobleroneSwordfish** [reply]
The way drones use the intent system is extremely unintuitive, but so are a lot of intent based things. Other than sticking it on the already overloaded flockmind and trace antag popups, I'm not really sure there's much we can do.
